feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -61,7 +62,7 @@ var (
 	}
 
 	atePort1 = attrs.Attributes{
-		Name:    "atePort1",
+		Name:    "port1",
```

Comment:
Why the change in name here?

Reply:
Restored the original names. I changed the name of the source port in the createFlow function, which is always "port1"